tests/scanners/test.binary.js

```diff
@@ -32,4 +34,7 @@ describe('Binary', function() {
     assert.equal(BinaryScanner.fileStreamType, 'chunk');
   });
 
+  it('should report a proper scanner name', () => {
+    assert.equal(BinaryScanner.scannerName, 'binary');
```

Comment:
Yeah, I think they could be constants. Let's do that though once we actually use them somewhere else.